### PR TITLE
Update Google Install and Release Notes with Core Bundle Note for 2.3.3

### PR DESCRIPTION
**Pull request by hickskenh**
## Purpose of this pull request

This pull request (PR) adds the core bundle note to the Install Guide and Release Notes documents for Google Shopping ads Channel. Also, the Add API Key steps are replaced in the Install Guide with a reference to the Merchdocs topic with the same information.

**These changes will not be merged until 2.3.3 is released.**

## Affected DevDocs pages

- https://devdocs.magento.com/extensions/google-shopping-ads/

- https://devdocs.magento.com/extensions/google-shopping-ads/release-notes/

### CODE REVIEW

File: extensions/google-shopping-ads/release-notes/index.md [hunk 1]
```diff
@@ -17,6 +21,15 @@ The release notes include:
 -   {:.fix}Fixes and improvements
 -   {:.bug}Known issues
 
+### V3.0.1
```
Comment (dshevtsov) [inline extensions/google-shopping-ads/release-notes/index.md]: There is inconsistency with the following heading. `V` vs `v`.

File: extensions/google-shopping-ads/release-notes/index.md [hunk 2]
```diff
@@ -17,6 +21,15 @@ The release notes include:
...
+
+{:.bs-callout .bs-callout-info}
+Beginning with Magento version 2.3.3, Google Shopping ads Channel is bundled and does not require separate install. Onboarding and management processes remain the same. Merchants using previous Magento versions (2.2.4+, 2.3.0, 2.3.1, and 2.3.2) can install or upgrade the Google Shopping ads Channel extension via the [Magento Marketplace](https://marketplace.magento.com/magento-google-shopping-ads.html).
+
+- {:.fix} **??**: ??.
```
Comment (dshevtsov) [inline extensions/google-shopping-ads/release-notes/index.md]: It is unclear what this should mean.

File: extensions/google-shopping-ads/release-notes/index.md [hunk 3]
```diff
@@ -17,6 +21,15 @@ The release notes include:
...
+
+- {:.fix} **??**: ??.
```
Comment (dshevtsov) [inline extensions/google-shopping-ads/release-notes/index.md]: It is unclear what this should mean.

File: extensions/google-shopping-ads/index.md [hunk 4]
```diff
@@ -3,13 +3,16 @@ group: extensions
 title: Install Google Shopping ads Channel
 ---
 
-The Google Shopping ads extension installs and adds Google Shopping features to Magento. To review additional information, see the [Google Shopping ads Channel Marketplace page](http://marketplace.magento.com/magento-google-shopping-ads.html).
+{:.bs-callout .bs-callout-info}
+Beginning with Magento version 2.3.3, Google Shopping ads Channel installs with the Magento update and does not require separate install. [Onboarding](https://docs.magento.com/m2/ee/user_guide/sales-channels/google-ads/onboarding-google.html) and management processes remain the same. Merchants using previous Magento versions (2.2.4+, 2.3.0, 2.3.1, and 2.3.2) can install or upgrade the Google Shopping ads Channel extension via the [Magento Marketplace](https://marketplace.magento.com/magento-google-shopping-ads.html).
+
+Google Shopping ads adds Google Shopping features to Magento. To review additional information, see the [Google Shopping ads Channel Marketplace page](http://marketplace.magento.com/magento-google-shopping-ads.html) and [Google Shopping ads Channel](https://docs.magento.com/m2/ee/user_guide/sales-channels/google-ads/google-ad-channel.html).
```
Comment (hguthrie) [inline extensions/google-shopping-ads/index.md]: ads adds? Tongue twisters on DevDocs! Maybe can say:
```suggestion
The Google Shopping ads channel integrates Google Shopping features with Magento. See [Google Shopping ads Channel Marketplace](http://marketplace.magento.com/magento-google-shopping-ads.html) and [Google Shopping ads Channel](https://docs.magento.com/m2/ee/user_guide/sales-channels/google-ads/google-ad-channel.html).
```

File: extensions/google-shopping-ads/release-notes/index.md [hunk 5]
```diff
@@ -3,7 +3,11 @@ group: extensions
 title: Google Shopping ads Channel Release Notes
 ---
 
-**Google Shopping ads Channel** is now generally available for versions 2.2.4+ and 2.3.x of {{site.data.var.ce}}, {{site.data.var.ee}}, and {{site.data.var.ece}}. Merchants can use this extension to integrate with Google Merchant Center (GMC) and Google Ads accounts to advertize and sell Magento products.
+The **Google Shopping ads Channel** extension is generally available for Magento versions 2.2.4+, 2.3.0, 2.3.1, and 2.3.2 of {{site.data.var.ce}}, {{site.data.var.ee}}, and {{site.data.var.ece}}. The extension can be installed or updated via the [Magento Marketplace](https://marketplace.magento.com/magento-google-shopping-ads.html).
```
Comment (hguthrie) [inline extensions/google-shopping-ads/release-notes/index.md]:
```suggestion
The **Google Shopping ads Channel** extension is available for Magento versions 2.2.4+, 2.3.0, 2.3.1, and 2.3.2 of {{site.data.var.ce}}, {{site.data.var.ee}}, and {{site.data.var.ece}}. The extension can be installed or updated using the [Magento Marketplace](https://marketplace.magento.com/magento-google-shopping-ads.html).
```

File: extensions/google-shopping-ads/release-notes/index.md [hunk 6]
```diff
@@ -17,6 +21,15 @@ The release notes include:
 -   {:.fix}Fixes and improvements
 -   {:.bug}Known issues
 
+### V3.0.1
+
+{:.bs-callout .bs-callout-info}
+Beginning with Magento version 2.3.3, Google Shopping ads Channel is bundled and does not require separate install. Onboarding and management processes remain the same. Merchants using previous Magento versions (2.2.4+, 2.3.0, 2.3.1, and 2.3.2) can install or upgrade the Google Shopping ads Channel extension via the [Magento Marketplace](https://marketplace.magento.com/magento-google-shopping-ads.html).
```
Comment (hguthrie) [inline extensions/google-shopping-ads/release-notes/index.md]:
```suggestion
Beginning with Magento version 2.3.3, Google Shopping ads Channel is bundled and does not require separate install. Onboarding and management processes remain the same. Merchants using previous Magento versions (2.2.4+, 2.3.0, 2.3.1, and 2.3.2) can install or upgrade the Google Shopping ads Channel extension using the [Magento Marketplace](https://marketplace.magento.com/magento-google-shopping-ads.html).
```

**Comment by hickskenh** [comment]
Closing this pull request.  the cbe language changes and install update were combined with the 3.0.1 release notes PR.

**Pull request closed without merging**